Add Defi Pulse Index (PoS)

Mringz opened the pull request:
Token Address: 0x85955046df4668e1dd369d2de9f3aeb98dd2a369
Token Name (from contract): DefiPulse Index (PoS)
Token Decimals (from contract): 18
Token Symbol (from contract): DPI
Link to the official homepage of token: https://www.indexcoop.com/
Link to CoinMarketCap or CoinGecko page of token: https://www.coingecko.com/en/coins/defipulse-index
Logo URL: https://gblobscdn.gitbook.com/assets%2F-MJY-enmfAw5ra2s-8QX%2F-MVfvTrtaK5Wt2GUktl-%2F-MVg-OcNO2oIxnjUsJTZ%2FDPI.jpg?alt=media&token=2731c21f-9d6f-4c98-b2a1-4761de7ea06b

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

Abhimanyu121 committed:
Added argo to the list

ss-sonic committed:
Merge pull request #4 from Abhimanyu121/main

Added argo to the list

polycatfi committed:
Update list-token.tokenlist.json

Add FISH

ss-sonic committed:
Merge pull request #5 from polycatfi/patch-1

Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

chandrazs committed:
Added Zee to Dfyn List

ss-sonic committed:
Merge pull request #6 from chandrazs/main

Added Zee to Dfyn List

ss-sonic committed:
Update list-token.tokenlist.json

Palette-Finance committed:
Add files via upload

Palette-Finance committed:
Merge pull request #1 from Palette-Finance/Palette-Finance-aurora

Add files via upload

ss-sonic committed:
Merge pull request #7 from Palette-Finance/main

AURORA,SOLAR

ss-sonic committed:
Update list-token.tokenlist.json

HikaruHokkyokusei committed:
Added Details of Matic Ruletka Token to the List.

ss-sonic committed:
Merge pull request #8 from HentaiOusama/patch-1

Added Details of Matic Ruletka Token to the List.

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

Palette-Finance committed:
Add files via upload

zhaojun-sh committed:
Add Anyswap token

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
patch list

syzonehq committed:
Added Unilend finance (UFT)

ss-sonic committed:
Merge pull request #12 from syzonehq/main

Added Unilend finance (UFT)

yun-yeo committed:
Add LUNA/UST token info

ss-sonic committed:
Merge pull request #13 from YunSuk-Yeo/patch-1

Add LUNA/UST token info

ss-sonic committed:
Update list-token.tokenlist.json

Akkii4 committed:
Added Cryption Network Token (CNT)

ss-sonic committed:
Merge pull request #14 from Akkii4/patch-1

Added Cryption Network Token (CNT)

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Added Ferrum

AGAToken committed:
Update list-token.tokenlist.json

ss-sonic committed:
Merge pull request #17 from AGAToken/patch-3

Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

RoyaleFinance committed:
Update list-token.tokenlist.json

ss-sonic committed:
Add AGA Tokens

ss-sonic committed:
Merge branch 'main' into patch-1

ss-sonic committed:
Merge pull request #18 from RoyaleFinance/patch-1

Update list-token.tokenlist.json

ss-sonic committed:
fix aga, agar

ss-sonic committed:
fix aga, agar

ss-sonic committed:
update token info

piratejeff committed:
feat: added pirate booty

piratejeff committed:
Fix: token address

ss-sonic committed:
Merge pull request #19 from piratejeff/main

[Feature] added pirate booty

ss-sonic committed:
Update list-token.tokenlist.json

InfernalHeir committed:
Whitelisting Unifarm Token

ss-sonic committed:
Merge pull request #23 from InfernalHeir/main

Whitelisting Unifarm Token

ss-sonic committed:
Update list-token.tokenlist.json

dobachtri committed:
Update list-token.tokenlist.json

Add ROSN token

dobachtri committed:
Update ROSN token

GabrielMCastro committed:
Add miMatic token to list-token.tokenlist.json

ss-sonic committed:
Merge pull request #24 from dobachtri/patch-1

Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Merge pull request #25 from GabrielMCastro/patch-1

Add miMatic token to list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

koinfoxcodebase committed:
Update list-token.tokenlist.json

Added Easy V2 (EZ) token

ss-sonic committed:
Merge pull request #26 from easyfinetwork/main

Add EASY V2(EZ)

ss-sonic committed:
Update list-token.tokenlist.json

xDollarFi committed:
add xUSD logo

Iamshashvat committed:
Updated tokenlist

Add DAIK, USDC, USDT, ETHK, BTCK

Iamshashvat committed:
Version updated

ss-sonic committed:
Merge pull request #29 from Iamshashvat/main

PR for adding new token in OKEx

ss-sonic committed:
Merge pull request #27 from xDollar-Finance/main

add xUSD logo

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

BeefyPower committed:
Update list-token.tokenlist.json

Add BIFI Polygon

ss-sonic committed:
Merge pull request #32 from BeefyPower/main

Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

Iamshashvat committed:
Titan token added

Iamshashvat committed:
Token updated and Version updated

ss-sonic committed:
Merge pull request #34 from dfyn/Iamshashvat-patch-2

Titan token added

saqib1421 committed:
Updation of RVF in list-token.tokenlist.json

saqib1421 committed:
Updated RVF image url from coingecko

ss-sonic committed:
Merge pull request #35 from saqib1421/main

Add Rocket Vault(RVF)

ss-sonic committed:
Update list-token.tokenlist.json

mindthecharts committed:
Update list-token.tokenlist.json

Added CHART token

NikolayNexo committed:
Added Nexo Details

ss-sonic committed:
Merge pull request #36 from mindthecharts/patch-1

Update list-token.tokenlist.json

ss-sonic committed:
Merge pull request #37 from NikolayNexo/add-nexo

Add Nexo Token

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

Gajesh2007 committed:
Added GAJ Token to Default DFYN List

0xMycaleum committed:
Update list-token.tokenlist.json

roelentless committed:
Add Attrace(ATTR) to list

ss-sonic committed:
Merge pull request #40 from attrace/main

Add Attrace(ATTR) to list

ss-sonic committed:
Merge pull request #39 from 0xMycaleum/Adding-Addy-to-Dfyn-whitelist

Adding Addy To Dfyn Whitelist

ss-sonic committed:
Update list-token.tokenlist.json

Gajesh2007 committed:
Merge branch 'main' into main

ss-sonic committed:
Merge pull request #38 from Gajesh2007/main

Added GAJ Token to Default DFYN List

elevenfinance committed:
Add ELE

ss-sonic committed:
Merge pull request #41 from elevenfinance/patch-2

Add ELE

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

hacker14398 committed:
Update the image address of Tokens

Booty, Nord, NWC, RBAL

ss-sonic committed:
Merge pull request #42 from dfyn/hacker14398-patch-1

Update the image address of Tokens

Iamshashvat committed:
Polytrade added

ss-sonic committed:
Merge pull request #44 from dfyn/Iamshashvat-patch-3

Polytrade added

Iamshashvat committed:
TRADE logo updated

ss-sonic committed:
Merge pull request #45 from dfyn/Iamshashvat-patch-4

TRADE logo updated

Iamshashvat committed:
Fix: Polytrade logo

Iamshashvat committed:
Merge pull request #46 from dfyn/Iamshashvat-patch-5

Fix: Polytrade logo

Iamshashvat committed:
REVERSE Token added

ss-sonic committed:
Merge pull request #47 from dfyn/Iamshashvat-patch-6

REVERSE Token added

aion-ollie committed:
Update list-token.tokenlist.json

ss-sonic committed:
Merge pull request #48 from aion-ollie/cirus

Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

oramis1008 committed:
update Iron stablecoin

oramis1008 committed:
undo format

Iamshashvat committed:
Merge pull request #49 from IronFinance/main

Adding IRON Stablecoin token

singularfarm committed:
Update list-token.tokenlist.json

Commit:
Added SILVER

Iamshashvat committed:
Merge pull request #52 from AFKSystemFi/patch-2

Added SILVER

Iamshashvat committed:
Merge pull request #50 from singularfarm/patch-1

Update list-token.tokenlist.json

ganeshbhagi committed:
add vDFYN

Iamshashvat committed:
Add MATRIX token

ss-sonic committed:
Merge pull request #53 from dfyn/Iamshashvat-patch-6

Add MATRIX token

ganeshbhagi committed:
update vDFYN address

Commit:
Added IRIS

Iamshashvat committed:
Merge pull request #54 from Hermes-defi/main

Added IRIS

finrenton committed:
Update Reverse 

Change to "logoURI": "https://i.imgur.com/AezG5si.png"

Iamshashvat committed:
Merge pull request #56 from finrenton/patch-4

Update Reverse

cyclese96 committed:
add PBR token

Iamshashvat committed:
Merge pull request #57 from cyclese96/patch-1

add PBR token

Iamshashvat committed:
Updated token List

 wFTM, DFYN, USDC, DAI, WBTC, fUSDT Token Added

Iamshashvat committed:
DFYN logo updated

ss-sonic committed:
Merge pull request #58 from dfyn/Iamshashvat-patch-7-1

Updated token List

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

ss-sonic committed:
Update list-token.tokenlist.json

Iamshashvat committed:
EMON token added

ss-sonic committed:
Merge pull request #59 from dfyn/Iamshashvat-patch-8

EMON token added

0xArata committed:
Apollo Token

0xArata committed:
Update list-token.tokenlist.json

ganeshbhagi committed:
Merge pull request #60 from poseidonOG/main

Apollo Token

ganeshbhagi committed:
add Apollo token and update version

ganeshbhagi committed:
Update list-token.tokenlist.json

ganeshbhagi committed:
Create README.md

ganeshbhagi committed:
Update README.md

arthurmsouza committed:
Add OMNI Token (OAI) Token

ss-sonic committed:
Merge pull request #61 from arthurmsouza/main

Add OMNI Token (OAI) Token

ss-sonic committed:
Update list-token.tokenlist.json

Iamshashvat committed:
SHIB Token added

ganeshbhagi committed:
Merge pull request #62 from dfyn/Iamshashvat-patch-8

SHIB Token added

ikramansari77 committed:
Update list-token.tokenlist.json

Iamshashvat committed:
Merge pull request #63 from ikramansari77/main

Add autonio (NIOX) to tokenlist

Iamshashvat committed:
SOL, MANA Added

ganeshbhagi committed:
Merge pull request #64 from dfyn/Iamshashvat-patch-9

SOL, MANA Added

kewlvishnu committed:
StackOS implementation on Polygon, DFYN

Iamshashvat committed:
Merge pull request #65 from kewlvishnu/patch-1

StackOS implementation on Polygon, DFYN

Iamshashvat committed:
version updated

ganeshbhagi committed:
Merge pull request #66 from dfyn/Iamshashvat-patch-10

version updated

Iamshashvat committed:
added testnet tokens

ganeshbhagi committed:
Merge pull request #67 from dfyn/Iamshashvat-patch-11

added testnet tokens

ss-sonic committed:
Update list-token.tokenlist.json

ARJUN-R34 committed:
Added SAFLE token data

ARJUN-R34 committed:
Updated token image URL

Iamshashvat committed:
Merge pull request #69 from getsafle/main

Added Safle token (SAFLE)

kridafans committed:
Update Krida Fans Token Details

Iamshashvat committed:
Merge pull request #70 from kridafans/main

Update Krida Fans Token Details

ganeshbhagi committed:
update version number

mynk-s-rwt committed:
Update list-token.tokenlist.json

mynk-s-rwt committed:
Updated chain ID for safle

Iamshashvat committed:
Merge pull request #71 from dfyn/Chota-Bheem-Codes-patch-1

Update list-token.tokenlist.json

Sushanthbc committed:
new host: Added coinwealth details

ganeshbhagi committed:
Merge pull request #72 from Sushanthbc/main

Added CoinWealth details

ganeshbhagi committed:
Update minor version 64 -> 65

0xvimal committed:
RVRS added

Iamshashvat committed:
Merge pull request #73 from dfyn/vimalrouter-patch-1

RVRS added

0xvimal committed:
Remove duplicate

Iamshashvat committed:
Merge pull request #74 from dfyn/vimalrouter-patch-1

Remove duplicate

balaianu committed:
Ad COR Token for ETH, BSC and Polygon chains

Iamshashvat committed:
Merge pull request #75 from coreto-io/main

Add COR Token for ETH, BSC and Polygon chains

Iamshashvat committed:
Update tokenlist

ganeshbhagi committed:
Merge pull request #76 from dfyn/Iamshashvat-patch-12

Update tokenlist

0xwrongchain committed:
Add UND (Unbound Dollar)

Iamshashvat committed:
Merge pull request #77 from 0xwrongchain/main

Add UND (Unbound Dollar)

Iamshashvat committed:
Update list-token.tokenlist.json

ganeshbhagi committed:
Merge pull request #78 from dfyn/Iamshashvat-patch-13

Update list-token.tokenlist.json

janeshbalchandani committed:
Added TitanDAO token

Iamshashvat committed:
Merge pull request #79 from dfyn/janeshbalchandani-patch-1

Added TitanDAO token

dirtycajunrice committed:
Add EVO token

ganeshbhagi committed:
Merge pull request #80 from dirtycajunrice/main

Add EVO token

dirtycajunrice committed:
Add EVO for additional networks

Iamshashvat committed:
Merge pull request #81 from dirtycajunrice/main

Add EVO for additional networks

janeshbalchandani committed:
Trava Finance (#82)

Commit:
SROCKET (Stable One Rocket) token details (#83)

ganeshbhagi committed:
update Minor version (#84)

janeshbalchandani committed:
O3 Swap Token update (#86)

janeshbalchandani committed:
UNB & UND Token update

Iamshashvat committed:
Merge pull request #87 from dfyn/janeshbalchandani-patch-4

UNB & UND Token update

mynk-s-rwt committed:
Fix Duplicate

Iamshashvat committed:
Merge pull request #88 from dfyn/Chota-Bheem-Codes-patch-1

Fix Duplicate

mynk-s-rwt committed:
Update GHST logouri

mynk-s-rwt committed:
Update minor version

Iamshashvat committed:
Merge pull request #89 from dfyn/Chota-Bheem-Codes-patch-2

Update GHST logouri

janeshbalchandani committed:
NORD Token added (#90)

* Update list-token.tokenlist.json

* NORD & ARTH Token

* NORD, MAHA, ARTH Tokens

vatsalgupta13 committed:
Removed duplicate NORD on 137

Iamshashvat committed:
Merge pull request #91 from dfyn/vatsalgupta13-patch-1

Removed duplicate NORD on 137

jdRouterP committed:
Add new tokens (#92)

* Update list-token.tokenlist.json

* Update

* update minor version and image of wbnb

* update wbnb image

janeshbalchandani committed:
Updated EasyFi token (#93)

* Updated EasyFi token

* Update list-token.tokenlist.json

janeshbalchandani committed:
EZ token on ETH (#94)

janeshbalchandani committed:
EZ on BNB (#95)

janeshbalchandani committed:
Gains Network Token Update (#97)

janeshbalchandani committed:
SEKUYA Token Update (#98)

Yakibanz commented:
"name": "Crypto 4 A Cause Fund",
 			"address": "0x8Fd0195469b51a935dC3C48617ced6B400E38C9C",
 			"symbol": "C4C",
 			"decimals": 18,
 			"chainId": 137,
 			"logoURI": "https://i.ibb.co/zSK16q3/c4c-token200.png"

"Dex Link": "https://app.dodoex.io/?network=polygon&from=USDT&to=C4C"
"Dextools Link": "https://www.dextools.io/app/en/polygon/pair-explorer/0x7daedc25c3d0bf505c4471908f8636df7d7156f4"

janeshbalchandani committed:
Gains Network Arbitrum Update (#99)

Iamshashvat committed:
Update token-list (#100)

hacker14398 committed:
Update TokenList (#102)

hacker14398 committed:
Update TokenList (#103)

mattyyapp committed:
Update list-token.tokenlist.json (#104)

* Update list-token.tokenlist.json

* Update list-token.tokenlist.json

mattyyapp committed:
Update list-token.tokenlist.json (#107)

mattyyapp committed:
Glitch Finance ETH + BSC token (#108)

* Update list-token.tokenlist.json

* Update list-token.tokenlist.json

mattyyapp committed:
Mummy Finance Optimism token (#109)

* Update list-token.tokenlist.json

* Update list-token.tokenlist.json

mattyyapp committed:
Mummy Finance Optimism token (#109)

* Update list-token.tokenlist.json

* Update list-token.tokenlist.json

mattyyapp committed:
Update list-token.tokenlist.json (#110)

kunalroute committed:
add: alpha dune token (#111)

* add: alpha dune token

* update: version

mattyyapp committed:
Update list-token.tokenlist.json (#112)

dev-cz committed:
Update list-token.tokenlist.json - Add Quick Intel Token (#114)

* Update list-token.tokenlist.json

Add Quick Intel to token list.

Name: Quick Intel
Symbol: QUICKI
Chain: 42161 (Arbitrum)
CA: 0x6d038130b9b379a373b1d33a29d5904ed1bb9026

* Update list-token.tokenlist.json

update minor version

---------

Co-authored-by: Ganesh <sivaganesh.bhagi@gmail.com>

ganeshbhagi committed:
Fix Update list-token.tokenlist.json (#115)

mattyyapp committed:
Update list-token.tokenlist.json (#116)

add $MMY on arbitrum

mynk-s-rwt committed:
Update list-token.tokenlist.json (#118)

* Update list-token.tokenlist.json

Add  Planq

* Update minor to 104

mattyyapp committed:
Update list-token.tokenlist.json (#119)

mattyyapp committed:
Update list-token.tokenlist.json (#120)

izi token 8 chains
